recipes/rodeo/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "0.4.4" %}
+
+package:
+  name: rodeo
+  version: {{ version }}
+
+source:
+  fn: rodeo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/r/rodeo/rodeo-{{ version }}.tar.gz
+  md5: cbd93b992b508eb6f0915214b3201661
+
+build:
+  number: 0
+  entry_points:
+    - rodeo = rodeo.cli:cmd
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - jupyter
+    - flask >=0.10.1
+    - docopt
+    - pyzmq >=13
+    - mistune
+    - gnureadline  # [osx]
+
+  run:
+    - python
+    - jupyter
```

Comment:
Does that really need the full jupyter stack or just jupyter-client? Using jupyter is probably "saver" for users which do not add environments, but for users which use environments this pulls in the complete stack which I have installed in multiple environments...